### DISCUSSION TITLE
Deepin File Manager not displaying image previews on remote mounts

### DISCUSSION THREAD

**Discussion by NjlsShade** [linuxdeepin/developer-center#3197] [Features Request & Ideas | 特性请求 & 头脑风暴]
### SUMMARY | 问题概要

When viewing a folder on a remote mount point the File Manager will not display image thumbnails. There should be a setting to enable Image previews on remote media.

### OBSERVED RESULT | 观察到的结果

Open a folder that contains images on a remote mount point, the images will not have previews.
 
![Screenshot_dde-file-manager_20220527213725](https://user-images.githubusercontent.com/15788214/170804897-e72e8c24-8660-4ff6-8e5d-90f29b7601fc.png)

### EXPECTED RESULT | 期望的结果

There should be a setting to allow image previews on remote mounts.

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

Using the latest deepin 20.5. File Manager 5.5.10.

### ADDITIONAL INFORMATION | 额外补充

For issues like this that are more of a basic feature request rather than a bug, should I report them in the issue tracker as I did here, or as a feature request discussion?

**Comment by justforlxz** [comment]
@meiyixiang

**Comment by BLumia** [comment]
I'll move feature requests to discussion board so we can focus on bugs on issue board.

If I remember correctly, the reason we don't add thumbnail to remote mounts before is the experience will be very bad if the connection is bad/slow. Since we need to fetch the content to local to perform the thumbnail generation process.

@meiyixiang : 文件管理器的网络共享那里，打开网络位置目前并不会加载缩略图，比如图片就会显示个图片的图标，而不是具体图片内容的缩略图。这里是希望能够显示能反映文件内容的缩略图。

**Comment by meiyixiang** [comment]
Thank you for your suggestions. I have recorded and fed back to the product, and hope to support it as soon as possible
